docs/promises.md

```diff
@@ -54,7 +54,7 @@ better stack traces.
 
 If you're an advanced user, you may want to plug in your own promise
 library like [bluebird](https://www.npmjs.com/package/bluebird). Just set
-`mongoose.Promise` to your favorite
+`global.Promise` to your favorite
```

Comment:
I'd recommend just removing this section of the docs entirely. We don't want to support or encourage overwriting `global.Promise`. If user does that, it's up to them.